packages/neo-one-node-core/src/StackItems/StackItems.ts

```diff
@@ -153,12 +152,12 @@ export const stackItemToJSON = (item: StackItem, context?: Set<StackItem>): Stac
     case StackItemType.Buffer:
       const buffer = assertBufferStackItem(item);
 
-      return { type: 'Buffer', value: JSONHelper.writeBuffer(buffer.getBuffer()) };
+      return { type: 'Buffer', value: JSONHelper.writeBase64Buffer(buffer.getBuffer()) };
```

Comment:
Buffer and ByteString are both written as base64 strings, not hex strings: https://github.com/neo-project/neo/blob/master/src/neo/VM/Helper.cs#L194